vNext/src/NuGet/Microsoft.Orleans.PlatformServices.nuspec

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Orleans.PlatformServices</id>
+    <version>$version$</version>
+    <title>Microsoft Orleans Platform Services</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft,Orleans</owners>
+    <projectUrl>https://github.com/dotnet/Orleans</projectUrl>
+    <licenseUrl>https://github.com/dotnet/Orleans#license</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/dotnet/orleans/gh-pages/assets/logo_128.png</iconUrl>
+    <summary>
+      Platform-specific functionality used by Microsoft Orleans - Orleans.PlatformServices.dll
+    </summary>
+    <description>
+      Provides platform-specific implementations of functionality required by Orleans.
+    </description>
+    <copyright>Copyright Microsoft 2016</copyright>
+    <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
+    <dependencies>
+      <group targetFramework=".NETStandard1.5">
+        <dependency id="System.Runtime.Loader" version="4.3.0" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="netstandard1.5\Orleans.PlatformServices.xml" target="lib\netstandard1.5" />
+    <file src="netstandard1.5\Orleans.PlatformServices.dll" target="lib\netstandard1.5" />
+    <file src="netstandard1.5\Orleans.PlatformServices.pdb" target="lib\netstandard1.5" />
+    <file src="net462\Orleans.PlatformServices.xml" target="lib\net462" />
+    <file src="net462\Orleans.PlatformServices.dll" target="lib\net462" />
+    <file src="net462\Orleans.PlatformServices.pdb" target="lib\net462" />
+    <file src="$SRC_DIR$Orleans.PlatformServices\**\*.cs" target="src" />
+  </files>
```

Comment:
Not really sure how it works, but shouldn't this have a `ref` folder? I'm basing this naively on `System.Runtime` package contents